opentok/endpoints.py

```diff
@@ -8,17 +8,17 @@ def __init__(self, api_url, api_key):
         self.api_url = api_url
         self.api_key = api_key
 
-    def session_url(self):
+    def get_session_url(self):
```

Comment:
Can we keep the old method names and mark them deprecated? They will just be wrappers for the new functions, but it gives users a chance to migrate to the new names before we do a breaking release (unless we've already got breaking changes waiting to go out, then the point is moot)

Reply:
Done